plc4go/examples/read/hello_world_plc4go_read.go

```diff
@@ -31,7 +31,7 @@ func main() {
 	drivers.RegisterModbusTcpDriver(driverManager)
 
 	// Get a connection to a remote PLC
-	crc := driverManager.GetConnection("modbus-tcp://192.168.23.30")
+	crc := driverManager.GetConnection("modbus-tcp://192.168.10.180")
```

Comment:
We should probably externalize this because every one is probably going to have the device at a different IP